library/cwm/src/lib/cwm/widget.rb

```diff
@@ -285,14 +285,27 @@ class CustomWidget < AbstractWidget
     abstract_method :contents
 
     def cwm_definition
-      res = { "custom_widget" => contents }
+      res = { "custom_widget" => cwm_contents }
 
       res["handle_events"] = ids_in_contents unless handle_all_events
 
       super.merge(res)
     end
 
-  private
+    # Returns all nested widgets used in contents
+    def nested_widgets
+      Yast.import "CWM"
+
+      Yast::CWM.widgets_in_contents(contents)
+    end
+
+  protected
+
+    def cwm_contents
```

Comment:
difference is that cwm_contents is content converted to format that CWM understand ( so widgets instance replaced by ids widget_id as string )